config.go

```
@@ -146,7 +173,8 @@ func (c *config) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 func (c *serverConfig) loadMatchers() error {
 	if len(c.MatchPatterns) == 0 {
-		return fmt.Errorf("no matchers specified")
+		log.Printf("INFO:  No matcher configured, fallback to name for %s", c.Name)
```

This seems unrelated to the change? We want to return an error here.